src/content/ui-api/management-api/authorization/index.md

```diff
@@ -0,0 +1,262 @@
+
+3. **Organization admin**: users who have full permissions to resources belonging to a specific [organization]({{< relref "/general/organizations" >}}).
+
+4. **Admin**: a type of user that has permission to create, modify, and delete most types of resources in the management cluster.
+
+Next we show how to configure access for these example cases. Please take into account these remarks for all example manifests:
+
+- You can set the resource names (for `RoleBinding` and `ClusterRoleBinding` resources) to whatever suits you and is available.
+- Make sure to replace `GROUPNAME` with the exact name of your group as defined in your identity provider.
+- Replace `ORGANIZATION` with the name of your organization (without `org-` prefix).
+- To assign a user instead of a group, replace `kind: Group` with `kind: User` in the subject entry and set `name` the exact email address of the users as set in your identity provider.
+
+### Configure access for read-only users {#configure-read-only}
+
+A read-only user in the sense of the Management API is one that can discover or, in the Giant Swarm web UI, browse resources in order to find out about existing clusters and their configuration, as well as apps installed and available.
+
+To enable a group or user for read access to resources of one organization, you only have to bind them to these two cluster roles within the namespace of the organization:
+
+- `read-all`
+- `read-in-cluster-ns`
+
+More details on these roles are given in the [pre-defined roles](#pre-defined-roles) section.
+
+The manifest below shows how the according role bindings could be created.
+
+```yaml
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: bind-read-all-to-group
+  namespace: org-ORGANIZATION
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: read-all
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: customer:GROUPNAME
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: bind-read-in-cluster-ns-to-group
+  namespace: org-ORGANIZATION
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: read-in-cluster-ns
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: customer:GROUPNAME
+```
+
+Our [web UI]({{< relref "/ui-api/web/organizations/access-control" >}}) makes it easy to create these role bindings interactively. Go to **Organizations**, select the organization, and navigate to the **Access control** tab. Then, one by one, select the pre-defined roles in the left column and add the user or group in the **Subjects** tab on the right.
+
+### Configure access for app developers {#configure-app-developer}
+
+Compared to a read-only user, this type of user would gain permission to install apps in workload clusters, change any app's configuration, and remove the app. To achieve this, instead of the `read-in-cluster-ns` role we bind `write-in-cluster-ns`.
```

Comment:
We should check on how this set of permissions needs to look like when the cluster namespace no longer exists.